Tests/Functional/StepProviders/CsvConfigurableStepsProviderTest.php

```diff
@@ -298,13 +298,13 @@ public function testAppendLinesToFile()
 
         //The original file should be there
         $this->assertFileExists(self::TMP_FOLDER.$fileName);
-        $lines = file(self::TMP_FOLDER.$fileName);
-        $this->assertEquals(count($lines), 7); //should be 7 rows
-        $this->assertEquals(count(explode('|', $lines[0])), 3); //should be 3 cols
+        $lines = \file(self::TMP_FOLDER.$fileName);
+        $this->assertEquals(\count($lines), 7); //should be 7 rows
```

Comment:
`$this->assertCount` would be a better choice

Reply:
I have tidied these up now